releasenotes/notes/use-gauge-metrics-in-rm-61ae620e83b95740.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    runtime_metrics: Adds support for sending runtime metrics as gauge metrics (instead of distributions). To enable tagging, set ``DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED=DD_RUNTIME_METRICS_ENABLED``. 
```

Comment:
```suggestion
    runtime_metrics: Adds support for sending runtime metrics as gauge metrics (instead of distributions). To enable this feature set ``DD_TRACE_EXPERIMENTAL_FEATURES_ENABLED=DD_RUNTIME_METRICS_ENABLED``. 
```